Networking/Networking/Model/ShippingLabel/Packages/WooShippingPackagesResponse.swift

```diff
@@ -1,12 +1,11 @@
 import Foundation
 import Codegen
 
-/// Represents store options, a list of saved Shipping Label Packages (custom and predefined) for the WooCommerce Shipping extension.
+/// Represents a list of available Shipping Label Packages (custom and predefined) for the WooCommerce Shipping extension.
 ///
 public struct WooShippingPackagesResponse: Equatable, GeneratedFakeable, GeneratedCopiable {
 
-    /// Store options
-    public let storeOptions: ShippingLabelStoreOptions
```

Comment:
I removed these since we're not using them right now, and it simplifies what we have to add to storage. We can always add them back later if we decide to use them.